### ISSUE TITLE
Some series won't play, may be due to subtitles?

### ISSUE THREAD

**Issue by Royster-77** [DxCx/plugin.video.wonderfulsubs#149]
Just thought I'd mention another issue I've noticed, some series like for example Shironeko Project ZERO Chronicle don't want to play in the plugin. It plays on the site but doesn't seem to have subtitles embedded, you have to select them before they'll show.  Not sure if that's relevant but from my non technical point of view It's the only difference I can see from series that did play.

**Comment by LordBoos** [comment]
I have this problem sometimes, but after several tries, the series will play. I noticed it on Overlord and Shield Hero.

**Comment by blunttedsmurf** [comment]
That anime is working for me with subs. I'm on the Firestick.

**Comment by Royster-77** [comment]
The first two worked for me too. Sorry should have been more specific it's episode 3 onwards, do they work for you?

**Comment by blunttedsmurf** [comment]
Yes they are working as well. Are you using the add-on with new url from this pull request? The old url stopped working and needs updating.

That's what I'm using until the official update comes out.

https://github.com/nalch/plugin.video.wonderfulsubs/tree/update-ws-api-url

![20200801_231733](https://user-images.githubusercontent.com/26640075/89114685-57dea200-d44d-11ea-8dfc-64dd49862c4f.jpg)
![20200801_231715](https://user-images.githubusercontent.com/26640075/89114686-590fcf00-d44d-11ea-802a-9eb2dc1b64d9.jpg)

**Comment by mrcheesedip** [comment]
> Yes they are working as well. Are you using the add-on with new url from this pull request? The old url stopped working and needs updating.
> 
> That's what I'm using until the official update comes out.
> 
> https://github.com/nalch/plugin.video.wonderfulsubs/tree/update-ws-api-url
> 
> ![20200801_231733](https://user-images.githubusercontent.com/26640075/89114685-57dea200-d44d-11ea-8dfc-64dd49862c4f.jpg)
> ![20200801_231715](https://user-images.githubusercontent.com/26640075/89114686-590fcf00-d44d-11ea-802a-9eb2dc1b64d9.jpg)

How did you get the updated addon with the new url onto the firestick?

**Comment by blunttedsmurf** [comment]
First remove the current Wonderfulsubs add-on then

1. On the Firestick open Downloader
3. Type this url: https://bit.ly/2PdjvFs
4. After download finishes, go back and open Kodi> System > Add-on
5. Do the install from zip and point to the zip in the Downloader folder

It should be good after that. You'll have to sign in again.

**Comment by Royster-77** [comment]
I've installed the above. It works with most other series now on the firestick and on my phone but that third ep still gives me an error on the firestick. Does work on my phone so not sure what I'm missing

**Comment by mrcheesedip** [comment]
> First remove the current Wonderfulsubs add-on then
> 
> 1. On the Firestick open Downloader
> 2. Type this url: https://bit.ly/2PdjvFs
> 3. After download finishes, go back and open Kodi> System > Add-on
> 4. Do the install from zip and point to the zip in the Downloader folder
> 
> It should be good after that. You'll have to sign in again.

Thank you this works! Can I ask, where did you find the link to download? I was searching for days to find out how to get it on firestick lol

**Comment by Royster-77** [comment]
Right...ok now Shironeko are all working.  No idea why they weren't before... 
Another series that I still can't get to play though is Shadowverse. It plays on the site, but checking on both the fire stick and my phone neither work through the plugin.  I get no eligible sources...

**Comment by Royster-77** [comment]
And having posted that then used a casting app to cast from the site on my phone to the firestick to watch ep 3.. the other episodes are working from the plugin. I guess it's the odd random ep here and there that don't want to play
